libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Storage/KeyValueEncryptedFileStore.swift

```diff
@@ -111,13 +111,17 @@ public class KeyValueEncryptedFileStore: NSObject {
         let versionFileURL = directory.appendingPathComponent(KeyValueEncryptedFileStore.storeVersionFileName)
         if isNewlyCreated {
             let versionFileCreated = writeFile(versionFileURL, content: KeyValueEncryptedFileStore.storeVersionString)
-            if !versionFileCreated {
+            let encryptionFileURL = directory.appendingPathComponent(KeyValueEncryptedFileStore.storeGCMEncryptionFileName)
```

Comment:
How is that file use?

Reply:
It's an empty file called "gcmEncryption", we're writing it upgrade on but I didn't do it for new stores. Having the legacy key will trigger the upgrade method but then the method will check for that file's existence to determine if the upgrade to gcm needs to happen or not

Reply:
👍